site/en/blog/inp-tools-2022/index.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Interaction to Next Paint (INP) Tool Support
```

Comment:
```suggestion
title: Interaction to Next Paint (INP) tool support
```

Reply:
> We use sentence case for titles. Also, should it be "tooling support" or is this sufficient?

I don't _hate_ the use of "tooling" but I do think people overuse it. Feel call me out if I'm overcorrecting, though :)